README.md

```diff
@@ -4,7 +4,17 @@ A plugin for IntelliJ IDEA that detects outdated comments in Java code
 
 ## Launch instructions
 
-- Inside the `src/main/kotlin/...models/config/ModelFilesConfig.kt` file 
-replace `DATA_DIR` value with your local path to the directory, containing files from [here](https://drive.google.com/drive/folders/1E4XOJHfEWOlHXBPxPu3_bkKfuU4G_N49?usp=sharing)
+### Plugin
+
+- Inside the `comment-updater-plugin/src/main/kotlin/...models/config/ModelFilesConfig.kt` file 
```

Comment:
1. Clone project `git clone https://github.com/JetBrains-Research/CommentUpdater.git`;
2. Download an [archive](https://drive.google.com/drive/folders/1E4XOJHfEWOlHXBPxPu3_bkKfuU4G_N49?usp=sharing) with model and code embeddings;
3. Open CommentUpdater project, go to `comment-updater-plugin/src/main/kotlin/...models/config/` and modify file `ModelFilesConfig.kt` by replacing `datadDir` value with the path to the directory containing files from Step 2;
4. Run `gradle/comment-updater-plugin/intellij/runIde` task.